translations/da.po

```diff
@@ -17064,13 +17064,13 @@ msgid "Date/time on which the article will be automatically unpublished"
 msgstr "Dato/tid hvor indholdet automatisk bliver afpubliceret."
 
 msgid "Event"
-msgstr "Begivenhed"
+msgstr "Arrangement"
 
 msgid "Category"
 msgstr "Kategori"
 
 msgid "Groups"
-msgstr "Grupper"
+msgstr "Tema"
```

Comment:
Please remember to add [an update hook to load the updated translations](https://github.com/ding2/ding2/blob/master/ding2.install#L598).

Reply:
~~Also: Are you sure that "Groups" (plural form) should be translated to "Tema" (singular form)?~~

I see that this is fine [as `ding_groups` actually calls the content type Groups](https://github.com/ding2/ding2/blob/e9b89ed2b14caa58d168a83a805fc3e03da5db6d/modules/ding_groups/ding_groups.features.inc#L73).